src/timeMachine/utils/rawFluxDataTable.ts

```diff
@@ -30,3 +37,96 @@ export const parseFiles = (responses: string[]): ParseFilesResult => {
 
   return {data, maxColumnCount}
 }
+
+export const parseFilesWithObjects = (
```

Comment:
uuuuugh tech debt

Reply:
Well, it is easy to pay.... should i mark it with a comment, something like, "Rip this out when RawDataView comes directly from Giraffe"?

Reply:
actually, probably not necessary, since the whole util file will be removed when the change to Giraffe's RawFluxDataTable happens.